pkg/vtime/time.go

```diff
@@ -0,0 +1,38 @@
+package vtime
```

Comment:
can you use jonboulle/clockwork instead? that's what we use in other CoreOS projects.